README.rst

```diff
@@ -56,7 +56,7 @@ To install PHP-CS-Fixer, install Composer and issue the following command:
 
     $ ./composer.phar global require fabpot/php-cs-fixer
 
-Then, make sure you have ``~/.composer/vendor/bin`` in your ``PATH``, and
```

Comment:
We do have docs guide, as mentioned [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1106#discussion_r27338111)